pages/common/mail.md

```diff
@@ -0,0 +1,32 @@
+# mail
+
+> The command operates on the user's mailbox if no argument is given.
+> To send an email the message body is built from standard input.
+
+- Send a typed email message. The commandline below continues after pressing Enter key. Input CC email-id (optional) press Enter key. Input message text (can be multi-line). Press "Ctrl-D" key to complete the message text:
+
+`mail --subject={{"subject line"}} {{to_user@example.com}}`
+
+- Send an email that contains file content:
+
+`mail --subject={{"$HOSTNAME filename.txt"}} {{to_user@example.com}} < {{path/to/filename.txt}}`
+
+- Send an email that contains command output:
```

Comment:
This is the same as the previous command. What's the difference ?

Reply:
I think the difference here is that it's meant to demonstrate that both a command can be executed and a file passed into the standard input. tldr-pages doesn't aim to replicate basic shell functionality in pages though as far as I'm aware, so we should keep just one of these 2 examples. Which one do you think we should keep?

Reply:
Let's keep the first one.